wolfhsm/wh_server.h

```diff
@@ -56,13 +56,18 @@ typedef struct CacheSlot {
 
 typedef struct {
     int devId;
-    Aes aes[1];
-    RsaKey rsa[1];
-    ecc_key eccPrivate[1];
-    ecc_key eccPublic[1];
-    curve25519_key curve25519Private[1];
-    curve25519_key curve25519Public[1];
     WC_RNG rng[1];
+    union {
+        Aes aes[1];
+        RsaKey rsa[1];
+        ecc_key eccPrivate[1];
+        curve25519_key curve25519Private[1];
+        Cmac cmac[1];
```

Comment:
Do these structs live beyond more than one client request?  If so, do we need another member that tracks which operation was pending so we can detect when the user is calling things out of sequence.

Or, if no state is necessary to be maintained between requests (except for RNG), then these should all be stack allocated during the request handling.